packages/js/product-editor/changelog/fix-product-editor-images-drop-error

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Show error notice when unable to upload an image.
```

Comment:
It seems that you need to add the `Product editor: ` prefix here?

Reply:
Hmm... I've never added the prefix before, since it is in the `@woocommerce/product-editor` package.